File: OPENAPI-REST-API/swagger-client/scala-akka-http-server/src/main/scala/io/swagger/server/model/QuickserverAddonsRow.scala
package io.swagger.server.model


/**
 * A single add-on item associated with a QuickServer service.
 *
 * @param desc Description for example: ''Additional IP Address''
 * @param value Value
 */
case class QuickserverAddonsRow (
  desc: Option[String],
  value: Option[String]
)

File: OPENAPI-REST-API/openapi-client/php-dt/src/App/DTO/QuickserverAddonsRow.php
<?php
declare(strict_types=1);

namespace App\DTO;

use Articus\DataTransfer\PhpAttribute as DTA;

/**
 * A single add-on item associated with a QuickServer service.
 */
class QuickserverAddonsRow
{
    /**
     * Description
     */
    #[DTA\Data(field: "desc", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $desc = null;

    /**
     * Value
     */
    #[DTA\Data(field: "value", nullable: true)]
    #[DTA\Validator("Scalar", ["type" => "string"])]
    public string|null $value = null;

}
